.azure-pipelines.yml

```diff
@@ -269,6 +269,34 @@ jobs:
 #################################################################################
 
 
+- job: 'linux_pypi_build'
+  displayName: 'Linux PyPI build'
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+  - checkout: self
+    submodules: true
+  - bash: |
+      set -eux
+      sudo apt install python python-pip
+      pip install twine
+    displayName: 'Bootstrap'
+  - bash: |
+      set -eux
+      python setup.py generate_pyqt sdist
+  - bash: |
+      echo $SNAPCRAFT_CREDENTIALS | twine upload dist/*
+    displayName: 'Build PyPI'
+    workingDirectory: $(Agent.TempDirectory)
+  - publish: $(Agent.TempDirectory)/dist
```

Comment:
What's the use for this ? (if the twine upload failed we stop before being here, if not we have the dist on pypi so it's easy to get them from there)

Reply:
I used the Snap code as a base so I kept it, as we already have the snaps in the artifacts, why not have the wheels too, just in case. But that's true, it is already on pypi, that's kind of useless...